binaries/runtime/Cargo.toml

```diff
@@ -32,7 +32,7 @@ tokio-stream = "0.1.8"
 zenoh = { git = "https://github.com/eclipse-zenoh/zenoh.git", rev = "79a136e4fd90b11ff5d775ced981af53c4f1071b" }
 zenoh-config = { git = "https://github.com/eclipse-zenoh/zenoh.git", rev = "79a136e4fd90b11ff5d775ced981af53c4f1071b" }
 fern = "0.6.1"
-pyo3 = { version = "0.16.5", features = ["auto-initialize", "eyre", "abi3"] }
+pyo3 = { version = "0.16", features = ["eyre", "abi3-py37"] }
```

Comment:
Doesn't this tie us to Python 3.7? What happens if the target system has e.g. 3.11 installed?

Reply:
This means, that the minimum version should be 3.7, but will work with 3.11 :)

Reply:
Ah, I skipped over the word "minimum" in https://pyo3.rs/latest/features.html#the-abi3-pyxy-features . Thanks for clarifying!